d8/d8batch/config/install/field.field.node.feed_source.field_feed_source_url.yml

```diff
@@ -0,0 +1,23 @@
+uuid: f29b03e1-489b-40ea-9a47-719dee675d3b
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_feed_source_url
+    - node.type.feed_source
+  module:
+    - link
+id: node.feed_source.field_feed_source_url
+field_name: field_feed_source_url
+entity_type: node
+bundle: feed_source
+label: URL
+description: ''
+required: true
+translatable: false
```

Comment:
Good question, all of these configuration files were exported with the Drupal CMI UI. I didn't wrote them manually.